go/kbnm/hostmanifest/whitelist_nix.go

```diff
@@ -0,0 +1,17 @@
+// +build !darwin
```

Comment:
Is there a convention for _suffix.go naming for "the rest"? In practice, this doesn't get used by Windows but it will get compiled nonetheless.